rpc/dataconn/timeoutconn/timeoutconn_readv_unsupported.go

```diff
@@ -0,0 +1,12 @@
+// +build illumos solaris
+
+package timeoutconn
+
+import (
+	"fmt"
+	"syscall"
+)
+
+func (c Conn) readv(rawConn syscall.RawConn, iovecs []syscall.Iovec) (n int64, err error) {
+	return 0, fmt.Errorf("Go does not support SYS_READV on this platform")
```

Comment:
would probably work if we used cgo and linked against the illumos C library, but that won't work with circleci
the fallback impl is really inefficient, though, because `iovecs` almost always contains two vecs, first big (for the last payload), second small (for the next header)